src/routing.js

```diff
@@ -72,7 +68,7 @@ export const parsePath = (pathname, search = '') => {
     }
   }
 
-  const [dao, instanceId, ...appParts] = parts
+  const [, instanceId, ...appParts] = parts
```

Comment:
Is this alright? there's a hanging comma in there

Reply:
Yep, this would skip the first element

Reply:
TIL, was not familiar with the syntax. 🍬 🍬

Reply:
Wow, magic